### DISCUSSION TITLE
docs/research/why-fish-dislike-work

### DISCUSSION THREAD

**Discussion by giscus** [MingGH/996-ninja-giscus#6] [General]
# docs/research/why-fish-dislike-work

根据法国科学家Mathias Pessiglione研究发现，思考时间越长，大脑中谷氨酸的含量也会逐步上升，而当谷氨酸的积累达到大脑规定阈值的时候，

https://996.ninja/docs/research/why-fish-dislike-work

**Comment by MingGH** [comment]
说实话，在[Mathias Pessiglione](https://sites.google.com/site/motivationbrainbehavior/the-team/current-team/pessiglione)的主页没找到类似研究，不知道是不是假的。。

**Reply by Mashirl** [reply]
我也好奇，所以用ChatGPT联网搜索试了一下，反馈如下：

我查阅了一些Mathias Pessiglione相关的研究文献，并没有找到直接表述“思考时间越长，大脑中谷氨酸的含量逐步上升，达到阈值后大脑会提高外侧前额叶的激活成本”的确切说法。不过，Pessiglione的研究确实涉及到大脑如何调节动机和决策过程，尤其是在奖励系统和前额叶功能方面。他的研究表明，大脑在高强度的决策和认知任务过程中会消耗一定的神经资源，可能导致疲劳和动机下降。

关于谷氨酸和大脑疲劳的具体机制，以下是几个可能相关的理论领域：

1. **神经递质与疲劳**  
   许多研究表明，**谷氨酸**是大脑中主要的兴奋性神经递质，参与认知功能和神经塑性过程。持续的认知负荷（如长时间思考或集中注意力）可能导致谷氨酸的积累，这会增加神经兴奋性，并可能对大脑的能量消耗产生影响。过度的兴奋性可能对大脑某些区域，尤其是**外侧前额叶皮层**的功能产生负面影响，从而导致疲劳和认知功能下降。

2. **前额叶皮层和疲劳的关系**  
   外侧前额叶皮层在**决策、注意力控制和任务管理**中起着至关重要的作用。长时间的认知任务可能导致这一区域的“激活成本”增加。研究发现，持续的脑力劳动会提高前额叶皮层的活动，增加大脑的能量消耗，从而导致**疲劳感**。

尽管没有找到与您提供的具体理论完全匹配的文献，但上述的神经科学理论有时可以从Pessiglione的研究中间接推导出相似的结论。以下是一些Pessiglione相关的论文，您可以进一步查阅，看看是否有相关的细节：

1. **Pessiglione, M., et al. (2007). "Dopaminergic Reward System and Decision-Making in the Human Brain."**  
   - 这篇文章探讨了大脑奖励系统与决策过程中的关系，但并未直接讨论谷氨酸和疲劳的机制。  
   - [PubMed链接](https://pubmed.ncbi.nlm.nih.gov/17656957/)

2. **Pessiglione, M., et al. (2013). "How the Brain Combines Choices and Rewards."**  
   - 本文详细讨论了奖励系统如何影响决策，涉及前额叶皮层的功能。  
   - [Science链接](https://www.science.org/doi/abs/10.1126/science.1231681)

3. **Pessiglione, M., et al. (2012). "Neural Mechanisms of Decision-Making and Motivation."**  
   - 这篇研究讨论了前额叶皮层在决策和动机中的作用，涉及神经递质在其中的作用。  
   - [Journal of Neuroscience链接](https://www.jneurosci.org/content/32/2/634)

您提到的关于谷氨酸积累与大脑疲劳的具体理论，可能是一些相关领域的研究总结或从其他来源得来的，但没有在Pessiglione的主流研究中找到直接的证据。如果您有更多的上下文或想继续探索，可以进一步查询更多关于神经递质和大脑疲劳的文献。

**Reply by MingGH** [reply]
![](https://img.996.ninja/ninjutsu/5d409a9cca0ca0c279d7f6154ecd4842.png)
